modules/uploader.js

```diff
@@ -55,19 +55,36 @@ class Uploader extends Module {
 
   upload(range, files) {
     const uploads = [];
-    Array.from(files).forEach(file => {
-      if (file && this.options.mimetypes.indexOf(file.type) !== -1) {
-        uploads.push(file);
+    const { mimetypes, handler, imageBlot } = this.options;
+
+    Array.from(files)?.forEach(file => {
+      if (file) {
+        let result = false;
+        mimetypes?.forEach(mimetype => {
+          const checkRegExpMimeType =
+            mimetype instanceof RegExp && mimetype.test(file.type);
+
+          if (checkRegExpMimeType || mimetype === file.type) {
+            result = true;
+            return false;
+          }
+
+          return false;
+        });
+
+        if (result) {
+          uploads.push(file);
+        }
       }
     });
     if (uploads.length > 0) {
-      this.options.handler.call(this, range, uploads, this.options.imageBlot);
+      handler.call(this, range, uploads, imageBlot);
     }
   }
 }
 
 Uploader.DEFAULTS = {
-  mimetypes: ['image/png', 'image/jpeg'],
+  mimetypes: [/^image\/(a?png|bmp|gif|p?jpe?g|svg|vnd\.microsoft\.icon|webp)/i],
```

Comment:
```suggestion
  mimetypes: [/^image\/(a?png|bmp|gif|p?jpe?g|svg|vnd\.microsoft\.icon|webp)/i],
```
Why? Just add additional mimetypes (as strings)